pkg/doc/verifiable/json_test.go

```diff
@@ -49,8 +49,9 @@ func Test_marshalJSON(t *testing.T) {
 
 	t.Run("Failed JSON marshall", func(t *testing.T) {
 		// artificial example - pass smth which cannot be marshalled
-		_, err := marshalWithCustomFields(make(chan int), map[string]interface{}{})
+		jb, err := marshalWithCustomFields(make(chan int), map[string]interface{}{})
```

Comment:
What does "jb" stand for?

Reply:
It's `jsonBytes`, that's easy, isn't it? :) just kidding, I will rename it!